Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -303,9 +316,75 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
             config.ScriptArguments = runTarget.Arguments;
             config.WorkingDirectory = runTarget.WorkingDirectory;
 
-            RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            if (useVTune) {
+                RunVTune(session, config, openReport);
+            } else {
+#endif
+                RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            }
+#endif
         }
 
+#if EXTERNAL_PROFILER_DRIVER
+        private static void RunVTune(SessionNode session, LaunchConfiguration config, bool openReport) {
+#if false
+            var pyexe = ProcessOutput.QuoteSingleArgument(stndTarget.InterpreterPath);
+
+            if (pyexe == string.Empty) {
+                if (stndTarget.PythonInterpreter != null) {
+                    var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
+                    var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
+                    if (interpreter == null) { 
+                        /* ??? */;
+                        MessageBox.Show("Could not find interpreter in the registry");
+                    } else {
+                        MessageBox.Show($"The Python interpreter in question is: [{interpreter}]");
+                    }
+                }
+            }
+#endif
+
+            var interpreter = config.GetInterpreterPath();
+            if (!File.Exists(interpreter)) {
+                MessageBox.Show("Can't find specified python interpreter.");
```

Comment:
Add to Strings.resx, and include the path in the message. Second param to Show should be Strings.ProductTitle.